test_data/makedata_suites/060_computed_values.js

```diff
@@ -0,0 +1,812 @@
+        `for doc in testView search doc.cv_field1=='foo' and doc.cv_field2=='bar' and doc.cv_field3=='baz' collect with count into c return c`,
+        `for doc in testView filter doc.cv_field == CONCAT(doc._key, ' ', doc._id, ' ', doc._rev) collect with count into c return c`,
+        `for doc in testViewV2 search doc.cv_field == SOUNDEX('sky') collect with count into c return c`, 
+        `for doc in testViewV2 search doc.cv_field == SOUNDEX('dog') collect with count into c return c`,
+        `for doc in testViewV2 search doc.cv_field_insert == SOUNDEX('frog') collect with count into c return c`,
+        `for doc in testViewV2 search doc.cv_field_update == SOUNDEX('beer') collect with count into c return c`,
+        `for doc in testViewV2 search doc.cv_field_replace == SOUNDEX('water') collect with count into c return c`,
+        `for doc in testViewV2 search doc.cv_field == null collect with count into c return c`,
+        `for doc in testViewV2 filter doc.cv_field == to_hex(doc.name) collect with count into c return c`,
+        `for doc in testViewV2 filter doc.cv_field == CONCAT('42_', TO_STRING(doc.field)) collect with count into c return c`,
+        `for doc in testViewV2 search doc.cv_field1=='foo' and doc.cv_field2=='bar' and doc.cv_field3=='baz' collect with count into c return c`,
+        `for doc in testViewV2 filter doc.cv_field == CONCAT(doc._key, ' ', doc._id, ' ', doc._rev) collect with count into c return c`
+      ]
+
+      let views_exp_output = [64000, 64000, 64000, 0, 0, 64000, 96000, 64000, 64000, 64000, 64000, 64000, 0, 0, 160000, 64000, 96000, 64000, 64000]
+
+      resultComparision(views_array, views_exp_output)
+
+
+      return 0;
+    },
+    clearDataDB: function (options, isCluster, isEnterprise, dbCount, database) {
+      print(`060: checking data ${dbCount}`);
+      let c1 = `c1_060_${dbCount}`;
+      let c2 = `c2_060_${dbCount}`;
+      let c3_insert = `c3_insert_060_${dbCount}`;
+      let c4_update = `c4_update_060_${dbCount}`;
+      let c5_replace = `c5_replace_060_${dbCount}`;
+      let c6_not_null = `c6_not_null_060_${dbCount}`;
+      let c7_hex = `c7_hex_060_${dbCount}`;
+      let c8_overwriteFalse = `c8_overwriteFalse_060_${dbCount}`;
+      let c9_overwriteTrue = `c9_overwriteTrue_060_${dbCount}`;
+      let c10_multiple = `c10_multiple_060_${dbCount}`;
+      let c11 = `c11_060_${dbCount}`;
+      let c12 = `c12_060_${dbCount}`;
+
+      Collection_array = [c1, c2, c3_insert, c4_update, c5_replace, c6_not_null, c7_hex, c8_overwriteFalse, c9_overwriteTrue, c10_multiple, c11, c12]
+
+      Collection_array.forEach(col => {
+        db.col.properties({computedValues: []})
+        //checking the properties set to null properly
+        if (db.col.properties()["computedValues"] === null) {
+          print('060: ok')
+          //drop the collection after check
+          db._drop(col);
+          progress(`deleting ${col} collection`);
```

Comment:
We will not see any errors or messages in case if `db.col.properties()["computedValues"] !== null`. Please change it and throw if computedValues are not deleted